### PR TITLE
Style: Reduced layout.css

### PR DESCRIPTION
**Pull request by CeltonMcGrath**

### CODE REVIEW

File: src/context/style/layout.css [hunk 1]
```diff
@@ -183,978 +154,4 @@ hr
...
-    opacity: 0.6;
-    display: none;
-}
-
-div.monitor {
-    padding: 2px;
-    margin: 4px 0 4px 0;
-}
-
-span.small
```
Comment (oboreo) [inline src/context/style/layout.css]: these span.small med large classes are used in diff places in the tactic source code.. What's the workaround if they are removed?

File: src/context/style/layout.css [hunk 2]
```diff
@@ -183,978 +154,4 @@ hr
...
-{
-    width: 14em;
-    float: left;
-} 
-
-div.lg_cell
-{
-    width: 24em;
-    float: left;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep for sm_cell, me_cell and lg_cell returns nothing.

File: src/context/style/layout.css [hunk 3]
```diff
@@ -183,978 +154,4 @@ hr
...
-    z-index: 3;
-    top: 0px;
-    left: 0px;
-    width: 100%;
-    height: 100%;
-    background: #999999;
-    filter: alpha(opacity=60); /* for IE compatibility, must occur before opacity */
-    opacity: 0.6;
-    display: none;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep for "overlay_dimmer" and "overlay_top" return no results.

File: src/context/style/layout.css [hunk 4]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-span.small
-{
-    padding: 0 5px 0 5px;
-}
-
-span.small_left
-{
-    padding-left: 5px;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: [tactic@localhost src]$ grep -r "small_left" .
Binary file ./pyasm/prod/web/asset_loader_wdg.pyc matches
./pyasm/prod/web/asset_loader_wdg.py:                td.add(SpanWdg(VersionWdg.get(status), css='small_left'))
Binary file ./pyasm/widget/discussion_wdg.pyc matches
./pyasm/widget/discussion_wdg.py:        status_span = SpanWdg(HtmlElement.b(status_value), css='small_left selection_item') 
./tactic/ui/cgapp/loader_element_wdg.py:                td.add(SpanWdg(VersionWdg.get(status), css='small_left'))
Binary file ./tactic/ui/cgapp/loader_element_wdg.pyc matches

File: src/context/style/layout.css [hunk 5]
```diff
@@ -183,978 +154,4 @@ hr
...
-    position: relative;
-}
-
-.clip_add span
-{
-    position: absolute; 
-    right: 2px; 
-    bottom: 0px;
-}
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: "clip_add" not found.

File: src/context/style/layout.css [hunk 6]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-
-
-/* TODO: not sure what this is for */
-.swap_link {
-    color: #E8C792;
-    cursor: pointer;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: [tactic@localhost src]$ grep -r "swap_link" .
./pyasm/widget/web_wdg.py:        #my.on_wdg.add_class("swap_link")
./pyasm/widget/web_wdg.py:        #my.off_wdg.add_class("swap_link")

File: src/context/style/layout.css [hunk 7]
```diff
@@ -183,978 +154,4 @@ hr
...
-/* Used for ActionSelectWdg .. primarily used for SimpleStatusWdg */
-select.action
-{
-    background-color: #dfe5bc;
-}   
-
-select.action_warning
-{
-    background-color: #f0d4af;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: [tactic@localhost src]$ grep -r "action_warning" .
./pyasm/widget/serial_status.py:                my.select.set_class('action_warning')

File: src/context/style/layout.css [hunk 8]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-/* defines a standard filter box on top of most TableWdg */
-div.filter_box
-{
-    border: 1px solid #aaa;
-    background-color: #eee;
-    margin: 5px 2px 5px 2px;
-    padding: 5px 2px 5px 2px;
-    /*text-align: center;*/
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: [tactic@localhost src]$ grep -r "filter_box" | wc -l
111

File: src/context/style/layout.css [hunk 9]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-div.background_box
-{
-    border: 1px solid #aaa;
-    background-color: #eee;
-    -moz-border-radius: 7px;
-    margin: 5px 5px 5px 5px;
-    padding: 5px 5px 5px 5px;
-
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "background_box"
Binary file pyasm/widget/clipboard_wdg.pyc matches
pyasm/widget/clipboard_wdg.py:            div.add_class("background_box")

File: src/context/style/layout.css [hunk 10]
```diff
@@ -183,978 +154,4 @@ hr
...
-{
-    height: 1.5em;   
-    -moz-border-radius: 6px;
-    border: 1px dotted #ccc; 
-    background-color: #dce0de;
-    padding: 6px 8px 6px 8px;
-    margin-bottom: -10px;
-    text-align: left;
-    font-weight: bolder;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "cell_header" | wc -l
0

File: src/context/style/layout.css [hunk 11]
```diff
@@ -183,978 +154,4 @@ hr
...
-  
-}
-*/
-
-
-img.icon_over {
-    filter: alpha(opacity=100); /* for IE compat. */
-    opacity: 1;
-}   
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: context/javascript/Common.js:        img_obj.className = 'icon_over';

File: src/context/style/layout.css [hunk 12]
```diff
@@ -183,978 +154,4 @@ hr
...
-img.icon_out {
-    filter: alpha(opacity=70); /* for IE compat. */
-    opacity: 0.7;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "icon_out"
context/javascript/Common.js:        img_obj.className = 'icon_out';
pyasm/widget/icon_wdg.py:            icon.add_class("icon_out")

File: src/context/style/layout.css [hunk 13]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-img.simple_button {
-    filter: alpha(opacity=70); /* for IE compat. */
-    opacity: 0.7;
-    cursor: pointer;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "simple_button"
context/spt_js/dg_table.js:    var img_list = table_top.getElements(".simple_button");
pyasm/widget/icon_wdg.py:            icon.add_class("simple_button")

File: src/context/style/layout.css [hunk 14]
```diff
@@ -183,978 +154,4 @@ hr
...
-div.discussion {
-    background-color: #f0f0f0;
-    font-size: 1.0em;
-    width: 350px;
-    -moz-border-radius: 9px;
-    padding: 6px 6px 6px 6px;
-    background-color: #f0f0f0;
-    text-align: left;
-    margin-bottom: 5px;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: This css is not being used.

File: src/context/style/layout.css [hunk 15]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-div.discussion_parent {
-    background-color: #d9d9d9;
-    margin: 3px 0px 3px 0px;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "discussion_parent"
Binary file pyasm/widget/discussion_wdg.pyc matches
pyasm/widget/discussion_wdg.py:                 new_note_swap, 'discussion_parent', use_notes)
pyasm/widget/discussion_wdg.py:        elif note_css == 'discussion_parent':

File: src/context/style/layout.css [hunk 16]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-div.discussion_sticky {
-    background-color: #E4F5C4;
-    margin: 3px 0px 3px 0px;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "discussion_sticky"
Binary file pyasm/widget/discussion_wdg.pyc matches
pyasm/widget/discussion_wdg.py:                 new_note_swap, 'discussion_sticky', use_notes)
pyasm/widget/discussion_wdg.py:        if note_css == 'discussion_sticky':

File: src/context/style/layout.css [hunk 17]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-div.discussion_child {
-    background-color: #c0c0c0;
-    margin: 3px 0px 3px 0px;
-}
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "discussion_child"
Binary file pyasm/widget/discussion_wdg.pyc matches
pyasm/widget/dependency_wdg.py:        #file_div = DivWdg(css='left_content discussion_child')
pyasm/widget/discussion_wdg.py:        elif note_css == 'discussion_child':
pyasm/widget/discussion_wdg.py:                     note_css='discussion_child')

File: src/context/style/layout.css [hunk 18]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-/* Trailing divs on the header */
-div.trail_on
-{
-    background-color: #e4e7eb;
-    width: 0.5em;
-    height: 0.5em;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "trail_on"
pyasm/widget/header_wdg.py:                    box.add_class('trail_on')

File: src/context/style/layout.css [hunk 19]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-div.trail_off
-{
-    background-color: transparent;
-    width: 0.5em;
-    height: 0.5em;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "trail_off"
pyasm/widget/header_wdg.py:                    box.add_class('trail_off')
Binary file pyasm/widget/header_wdg.pyc matches

File: src/context/style/layout.css [hunk 20]
```diff
@@ -183,978 +154,4 @@ hr
...
-div.annotate_new {
-    position: absolute;
-    background-color: white;
-    border-style: solid;
-    border-color: black;
-    border-width: 1px;
-    z-index: 6;
-    padding: 3px;
-    width: 230;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "annotate_new"
Binary file pyasm/widget/annotate_wdg.pyc matches
pyasm/widget/annotate_wdg.py:        new_annotation_div.set_class("annotate_new")

File: src/context/style/layout.css [hunk 21]
```diff
@@ -183,978 +154,4 @@ hr
...
-    margin-top: 0.5px;
-    padding: 10px 10px 10px 10px;
-}
-
-.tab_sm_content {
-    background-color: #b9cac8;
-    border: #8cacbb groove 2px;
-    margin-top: -0.5px;
-    padding: 10px 10px 10px 10px;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "tab_sm_content" | wc -l
0

File: src/context/style/layout.css [hunk 22]
```diff
@@ -183,978 +154,4 @@ hr
...
-    padding: 0px 3px;
-    border-spacing: 0px;
-}
-
-.minimal_header td, .minimal_header th {
-    font-size: 0.7em;
-    font-weight: 400;
-    background-color: #fff;
-    height: 1.0em;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "minimal_header" | wc -l
0

File: src/context/style/layout.css [hunk 23]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-tr.plain_bg td
-{
-    background-color: #e3e8e6;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "plain_bg"
pyasm/widget/sobject_group_wdg.py:        main_table.add_row(css='plain_bg')
Binary file pyasm/widget/sobject_group_wdg.pyc matches
tactic/ui/widget/sobject_group_wdg.py:        main_table.add_row(css='plain_bg')
Binary file tactic/ui/widget/sobject_group_wdg.pyc matches

File: src/context/style/layout.css [hunk 24]
```diff
@@ -183,978 +154,4 @@ hr
...
-    width: 50px;
-    -moz-border-radius: 4px;
-}
-
-div.slide_panel_cont {
-    position: absolute;
-    left: 0px;
-    top: 10px;
-    z-index: 500;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "slide_panel_cont"
Binary file pyasm/widget/web_wdg.pyc matches
pyasm/widget/web_wdg.py:    SLIDE_PANEL_CONTAINER = 'slide_panel_cont'

File: src/context/style/layout.css [hunk 25]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-/* Theme: DARK */
-
-/* --- NavMenu look styles (for Side Bar Nav) --- */
-
-.look_navmenu_header {
-    background-color: #c2c2c2;
-    color: #393950;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_navmenu_header" | wc -l
0

File: src/context/style/layout.css [hunk 26]
```diff
@@ -183,978 +154,4 @@ hr
...
-.look_navmenu_section {
-    background-color: #949494;
-    color: #292940;
-}
-
-.look_navmenu_scroll {
-    background-color: #656565;
-}
-
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_navmenu_" | wc -l
0

File: src/context/style/layout.css [hunk 27]
```diff
@@ -183,978 +154,4 @@ hr
...
-    color: #c2c2d2;
-    border: 1px solid #000000;
-}
-
-.look_popup_dragbar {
-    background-color: #202020;
-    color: #c2c2d2;
-    cursor: move;
-    border-bottom: 1px solid #000000;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_popup" | wc -l
0

File: src/context/style/layout.css [hunk 28]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-/* --- Page look styles --- */
-
-.look_page {
-    background-color: #434343;
-    color: #c2c2c2;
-    border-color: #c2c2c2;  /* also used for HR elements */
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_page" | wc -l
0

File: src/context/style/layout.css [hunk 29]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-.look_mms_dg_row_selected {
-    background-color: #4ea0d9;
-}
-
-
-.look_mms_dg_row_selected_hilite {
-    background-color: #6ec0f9;
-}
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_mms" | wc -l
0

File: src/context/style/layout.css [hunk 30]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-/* value changed look styles ... */
-
-.look_dg_row_value_changed {
-    background-color: #002200;
-}
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_row_value_changed" | wc -l
0

File: src/context/style/layout.css [hunk 31]
```diff
@@ -183,978 +154,4 @@ hr
...
-.look_dg_field_value_changed {
-    background-color: #909977;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_field_value_changed" | wc -l
0

File: src/context/style/layout.css [hunk 32]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-/* smenu look styles -- dark text on light background version ... */
-
-.look_smenu {
-    border-color: #000000;
-    background-color: #c2c2c2;
-    color: #000000;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_smenu"
Binary file tactic/ui/panel/layout_wdg.pyc matches
tactic/ui/panel/layout_wdg.py:        text.add_class('look_smenu_disabled')

File: src/context/style/layout.css [hunk 33]
```diff
@@ -183,978 +154,4 @@ hr
...
-    color: #656565;
-    background-color: #656565;
-    height: 1px;
-}
-
-.look_smenu_title {
-    color: #c2c2c2;
-    background-color: #434343;
-}
-
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: see comment above, most look_smenu can be removed.

File: src/context/style/layout.css [hunk 34]
```diff
@@ -183,978 +154,4 @@ hr
...
-    cursor: default;
-}
-
-.look_dg_row_select_box_selected
-{
-    background-image: url('/context/ui_proto/icons/interface/checkbox_checked.png');
-    background-position: center center;
-    background-repeat: no-repeat;
-    cursor: default;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_select_box_selected" | wc -l
5

File: src/context/style/layout.css [hunk 35]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-.look_dg_row_select_box
-{
-    background-image: url('/context/ui_proto/icons/interface/checkbox_unchecked.png');
-    background-position: center center;
-    background-repeat: no-repeat;
-    cursor: default;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_select_box" | wc -l
10

File: src/context/style/layout.css [hunk 36]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-.look_dg_row_selected {
-    background-color: #292940;
-}
-
 
-.look_dg_row_selected_hilite {
-    background-color: #393950;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_selected_hilite" | wc -l
0

File: src/context/style/layout.css [hunk 37]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-.look_dg_row {
-    background-color: #434343;
-}
-
-
-.look_dg_row_hilite {
-    background-color: #535353;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_hilite" | wc -l
0

File: src/context/style/layout.css [hunk 38]
```diff
@@ -183,978 +154,4 @@ hr
...
-    background-color: #761010;
-}
-
-.look_dg_row_retired_selected {
-    background-color: #330000;
-}
-
-.look_dg_row_retired_selected_hilite {
-    background-color: #431010;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_retired_selected_hilite" | wc -l
0

File: src/context/style/layout.css [hunk 39]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-.look_dg_row {
-    background-color: #434343;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row"
context/spt_js/dg_table.js:    var row_is_retired = activator_el.hasClass('look_dg_row_retired');
tactic/ui/panel/table_layout_wdg.py:        if (cell.hasClass("look_dg_row_select_box") ) {
tactic/ui/panel/table_layout_wdg.py:            cell.addClass("look_dg_row_select_box_selected");
tactic/ui/panel/table_layout_wdg.py:            cell.removeClass("look_dg_row_select_box");
tactic/ui/panel/table_layout_wdg.py:            cell.removeClass("look_dg_row_select_box_selected");
tactic/ui/panel/table_layout_wdg.py:            cell.addClass("look_dg_row_select_box");
tactic/ui/panel/table_layout_wdg.py:        cell.removeClass("look_dg_row_select_box");
tactic/ui/panel/table_layout_wdg.py:        cell.addClass("look_dg_row_select_box_selected");
tactic/ui/panel/table_layout_wdg.py:        cell.removeClass("look_dg_row_select_box_selected");
tactic/ui/panel/table_layout_wdg.py:        cell.addClass("look_dg_row_select_box");

File: src/context/style/layout.css [hunk 40]
```diff
@@ -183,978 +154,4 @@ hr
...
-    opacity: .5;
-    filter: alpha(opacity=50);
-}
-
-
-/* dg table look styles */
-
-.look_dg_header_cell_hilite {
-    background-color: #292940;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_header_cell_hilite" | wc -l
0

File: src/context/style/layout.css [hunk 41]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-.look_dg_row_retired {
-    background-color: #660000;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_retired"
context/spt_js/dg_table.js:    var row_is_retired = activator_el.hasClass('look_dg_row_retired');

File: src/context/style/layout.css [hunk 42]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-.look_dg_row_retired_hilite {
-    background-color: #761010;
-}
-
-.look_dg_row_retired_selected {
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_retired_selected" | wc -l
0

File: src/context/style/layout.css [hunk 43]
```diff
@@ -183,978 +154,4 @@ hr
...
-    background-color: #292940;
-}
-
-.look_dg_row_retired {
-    background-color: #660000;
-}
-
-.look_dg_row_retired_hilite {
-    background-color: #761010;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_retired_hilite" | wc -l
0

File: src/context/style/layout.css [hunk 44]
```diff
@@ -183,978 +154,4 @@ hr
...
-
-
-.look_dg_row_hilite {
-    background-color: #535353;
-}
-
-
-.look_dg_row_selected {
-    background-color: #292940;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_dg_row_selected" | wc -l
0

File: src/context/style/layout.css [hunk 45]
```diff
@@ -183,978 +154,4 @@ hr
...
-.look_dg_field_value_changed {
-    background-color: #909977;
-}
-
-
-/* input look styles ... */
-
-.look_input_validation_failed {
-    background-color: #cc3333;
-}
```
Comment (CeltonMcGrath) [inline src/context/style/layout.css]: grep -r "look_input_validation_failed" | wc -l
0

**Comment by remkonoteboom** [comment]
Closing this pull request.  This has all been done in 4.6.  Been running for a long time with a greatly reduced css file with no discernible issues ... will fix on fail if anything comes up.

**Pull request closed without merging**